fractal_client/cmd/_group.py

```diff
@@ -0,0 +1,56 @@
+from ..authclient import AuthClient
+from ..config import settings
+from ..interface import Interface
+from ..response import check_response
+
+
+def group_list(
+    client: AuthClient, *, user_ids: bool = False, batch: bool = False
+):
+    query_params = "?user_ids=true" if user_ids else ""
+    res = client.get(f"{settings.FRACTAL_SERVER}/auth/group/{query_params}")
+    data = check_response(res, expected_status_code=200)
+    if batch:
+        return Interface(
+            retcode=0, data=" ".join([str(d["id"]) for d in data])
+        )
+    else:
+        return Interface(retcode=0, data=data)
+
+
+def group_get(client: AuthClient, *, group_id: int, batch: bool = False):
```

Comment:
Let's remove the `batch` option from `group_get`, since it is not very meaningful for `fractal group show 99` to return `99`